README.md

```diff
@@ -217,6 +217,61 @@ We need to first add patient's details into wrapper's database. There are differ
          }
      ]'
      ```
+### Profile Share
+Adding of a patient in a particular facility
+- When the user scans and shares their profile, the wrapper generates a token and sends to the facility for notifying about the patient
+  - So the facility have to implement
+  * `POST` Request `/v1/profile/share`
+  ```
+    {
+      "token":"Wrapper generated token number for the user"
+      "hipId":"the HIP's counter code not the hip id"
```

Comment:
is this needed? since you already have hipCode in profile

Reply:
The code in the profile is the facility's counter code, the exact hipId comes in the headers of the api.